core/services/pg/utils.go

```diff
@@ -22,9 +22,17 @@ const (
 
 var _ driver.Valuer = Limit(-1)
 
+var _ QConfig = &qConfig{}
+
 // Limit is a helper driver.Valuer for LIMIT queries which uses nil/NULL for negative values.
 type Limit int
 
+// qConfig implements pg.QCOnfig
+type qConfig struct {
+	logSQL              bool
+	defaultQueryTimeout time.Duration
+}
```

Comment:
Let's keep all of these pieces co-located in one block, instead of spread through the file like this.